lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -51,7 +51,6 @@ def build_launch_request(authenticated=True):
     """
     request = RequestFactory().post('/')
     request.user = UserFactory.create()
-    request.user.is_authenticated = MagicMock(return_value=authenticated)
```

Comment:
It looks like you should be able to remove the `authenticated` function argument now.